install/upgrade/update_to_dev.php

```diff
@@ -119,25 +122,29 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $migration->addKey($table, 'entities_id', 'entities_id');
    $migration->addPostQuery("UPDATE `$table` SET `parse_status` = 'parsed'");
 
-   $migration->addKey('glpi_plugin_flyvemdm_files', 'entities_id', 'entities_id');
-   $migration->addKey('glpi_plugin_flyvemdm_fleets', 'entities_id', 'entities_id');
-   $migration->addKey('glpi_plugin_flyvemdm_geolocations', 'computers_id', 'computers_id');
-   $migration->addKey('glpi_plugin_flyvemdm_mqttacls', 'plugin_flyvemdm_mqttusers_id',
-      'plugin_flyvemdm_mqttusers_id');
-   $migration->addKey('glpi_plugin_flyvemdm_policycategories',
-      'plugin_flyvemdm_policycategories_id', 'plugin_flyvemdm_policycategories_id');
-   $migration->addKey('glpi_plugin_flyvemdm_invitationlogs', 'glpi_plugin_flyvemdm_invitationlogs',
-      'invitations_id');
+   $table = 'glpi_plugin_flyvemdm_files';
+   $migration->addKey($table, 'entities_id', 'entities_id');
+
+   $table = 'glpi_plugin_flyvemdm_fleets';
+   $migration->addKey($table, 'entities_id', 'entities_id');
+
+   $table = 'glpi_plugin_flyvemdm_geolocations';
+   $migration->addKey($table, 'computers_id', 'computers_id');
+
+   $table = 'glpi_plugin_flyvemdm_mqttacls';
+   $migration->addKey($table, 'plugin_flyvemdm_mqttusers_id', 'plugin_flyvemdm_mqttusers_id');
+
+   $table = 'glpi_plugin_flyvemdm_policycategories';
+   $migration->addKey($table, 'plugin_flyvemdm_policycategories_id', 'plugin_flyvemdm_policycategories_id');
+
+   $table = 'glpi_plugin_flyvemdm_invitationlogs';
+   $migration->addKey($table, 'glpi_plugin_flyvemdm_invitationlogs', 'invitations_id');
 
    $table = 'glpi_plugin_flyvemdm_invitations';
    $migration->addKey($table, 'users_id', 'users_id');
    $migration->addKey($table, 'entities_id', 'entities_id');
    $migration->addKey($table, 'documents_id', 'documents_id');
 
-   $table = 'glpi_plugin_flyvemdm_taskstatuses';
-   $migration->addKey($table, 'plugin_flyvemdm_agents_id', 'plugin_flyvemdm_agents_id');
-   $migration->addKey($table, 'plugin_flyvemdm_tasks_id', 'plugin_flyvemdm_tasks_id');
```

Comment:
Are not necessary this keys?, the update_to_2_0_0.php file has them.

Reply:
actually I grouped the changes by table.
* $table affectation moved to L82
* addkey() calls moved to L98

Reply:
ok then it looks good for me!.